lms/templates/main.html

```diff
@@ -59,10 +60,16 @@
 
   <script type="text/javascript" src="/jsi18n/"></script>
 
-  <link rel="icon" type="image/x-icon" href="${static.url(microsite.get_value('favicon_path', settings.FAVICON_PATH))}" />
+  <link rel="icon" type="image/x-icon" href="${static.url(settings.FAVICON_PATH)}" />
```

Comment:
This loses microsites.  Whatever happened to the per-microsite settings pull request?